run-tests-jenkins

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install coverage
```

Comment:
Yes, this can be part of the outer Dockerfile as a new line: `RUN pip install coverage`.
At some point the accumulated new outer Dockerfile commands get migrated to the inner Dockerfile for hs_base and compiled as part of the pulled base image.